src/main/java/org/apereo/openequella/adminconsole/service/JarService.java

```diff
@@ -136,7 +136,11 @@ public void downloadJar(String jarName) {
 	public int executeJar(String jarName, String mainClass, String... jvmArgs) {
 		final File binJar = getBinJarFile(jarName);
 		final List<String> command = new ArrayList<>();
-		command.add("java");
+		// Use the bundled JRE to run admin console
+		final String os = System.getProperty("os.name");
+		final String bundledJrePath = System.getProperty("user.dir")+"/jdk8u212-b03-jre/";
```

Comment:
Could you change to:
```
File currentPath = new File(System.getProperty("user.dir"));
File bundledJrePath = new File(currentPath, "jdk8u212-b03-jre");
File binPath = new File(bundledJrePath, os.toLowerCase().indexOf("mac") > 0 ? "Contents/Home/bin/" : "bin");
String fullJavaPath = new File(binPath, "java").getAbsolutePath();
```
It's a bit more wordy, but you don't have to deal with \ or / path separators